### PR TITLE
Added wishlist functionality

### LINKED ISSUE #22: Added wishlist functionality
Added wishlist page
User can move product to cart
user can delete the product from wishlist

### PR DESCRIPTION
**Pull request by Raghav888**
1. Added wishlist page
2. User can move product to cart
3. user can delete the product from wishlist

### CODE REVIEW

**Review by SuyashTiwariCoder — APPROVED** [review]
code looks good , and well organised , good to go  .

File: src/components/detailcart/Detailcart.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ export const Detailcart = () => {
   const { cartState, cartDispatch } = useCart();
   const { productListdispatch } = useProductList();
   const { wishlistState, wishlistDispatch } = useWishlist();
-
+  console.log(cartState.cartData);
```
Comment (mariyasada) [inline src/components/detailcart/Detailcart.js:11]: remove console

**Review by mariyasada — APPROVED** [review]
very well structured code. liked that you have created a separate context for the cart and wishlist

**Review by paarthbajaj — APPROVED** [review]
I didn't find any place for improvement maybe because I am still learning but I like how you managed to keep your code clean and readable and proper choice of conditional statements like using ternary and correct usage of ES6.

**Pull request merged**